app/configurator/configurator-state.tsx

```diff
@@ -770,48 +766,47 @@ const reducer: Reducer<ConfiguratorState, ConfiguratorStateAction> = (
       }
       return draft;
 
-    case "CHART_CONFIG_FILTER_SET_MULTI":
-      if (draft.state === "CONFIGURING_CHART") {
-        const { dimensionIri, value } = action.value;
-        draft.chartConfig.filters[dimensionIri] = {
-          type: "multi",
-          values: { [value]: true },
-        };
-      }
-      return draft;
-
     case "CHART_CONFIG_FILTER_ADD_MULTI":
       if (draft.state === "CONFIGURING_CHART") {
-        const { dimensionIri, value, allValues } = action.value;
+        const { dimensionIri, values, allValues } = action.value;
         const f = draft.chartConfig.filters[dimensionIri];
+        const valuesUpdate = Object.fromEntries(
+          values.map((v: string) => [v, true as true])
```

Comment:
Is this `true as true` done on purpose?

Reply:
I am going to check but I think yes since otherwise it is considered as boolean whereas here there can't be `false`.

Reply:
This is because of the type of the filters in chartConfig:

```
{
        type: "multi";
        values: {
            [x: string]: true;
        };
    }
```